src/Command/CreateClassCacheCommand.php

```diff
@@ -70,7 +70,7 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $output->write('<info>Writing cache file ...</info>');
         ClassCollectionLoader::load(
-            include($classmap),
+            include ($classmap),
```

Comment:
This conflict with phpcs fixer

Reply:
I created a new php-cs-fixer 2.17.0 release in the morning